flint/summary.py

```diff
@@ -89,6 +92,26 @@ def with_options(self, **kwargs) -> FieldSummary:
         return FieldSummary(**prop)
 
 
+def _get_pol_axis_as_rad(ms: Union[MS, Path]) -> float:
```

Comment:
This is the function that attempts to get the most correct orientation. I am not sure whether I want to force an error to be raised if the instrument receptor angle is missing, or whether this is ok. This is a non-standard keyword.

Reply:
Depends on how much we want to be locked into FixMS. Will the observatory MSs ever have correct feeds?

Reply:
At this point I would not expect them to.

If they did I would they would also rotate the correlations to the sky as well as update the receptor angle. if so, I think the problem kind of drops out even if we keep fixms in out respective codez.